topics/materials-science/tutorials/muspinsim/workflows/1_Zeeman_Coupling.ga

```diff
@@ -0,0 +1,174 @@
+{
```

Comment:
Yeah don't know if I did this or if it was already present in the workflow. Personally I prefer reduced clutter, and to me I don't see the config file as a "meaningful" output, all it's doing is transcribing what I set in the UI so I shouldn't need to check the output (versus a report on a process which has a non-deterministic output, or could fall over during operation). The [GTN says](https://usegalaxy.org/training-material/topics/galaxy-interface/tutorials/history/tutorial.html#managing-datasets-individually) "_workflows will often hide history datasets in order to simplify the history and hide intermediate steps of an automated analysis_", so I reckon we're within the intended use case (and teaching people about hidden datasets is not the remit of our tutorial).

I don't feel strongly though. I don't know if @joelvdavies 's opinion would be useful here as a someone who might not be as familiar with histories and hiding datasets?

Reply:
I didn’t realise it was hiding anything. I felt the tutorial was clear enough that I knew the steps needed to obtain the output regardless of whether these are hidden or not. I too think it would be nicer to reduce clutter in this case as it could feel like we're overloading the user with information.

Reply:
Sure, leave them as they are then. I think the difference felt stronger because I ran the steps manually and the workflow immediately after, plus I have that full knowledge of what's happening.
Worth noting that the AIRSS workflow gives you all the intermediate outputs currently, but we could very well hide all those zip files too for the same reason.

Reply:
Didn't realise this but Galaxy doesn't really seem to support hidden collections - while you can hide them in workflows you can't hide them manually from your history, and if they are hidden then you can't select them until you go in and manually unhide them. So I'm going to make the intermediary collections visible for all the workflows (currently it is for some, isn't for others).